api/src/messaging/consumers.py

```diff
@@ -37,6 +45,10 @@ async def disconnect(self, close_code):
     async def receive(self, text_data=None, byte_data=None):
         text_data_json = json.loads(text_data)
         user_id = self.scope["user"].id
+        if not self.room:
+            # panic error?
+            pass
+
```

Comment:
Why do we need this?

Reply:
I thought I had a good reason when I was writing this, but now I can't see it; removed